src/test/data/unified_addrs.json

```diff
@@ -1,14 +1,14 @@
 [
```

Comment:
I verified these changes matched the test vectors generated by https://github.com/zcash-hackworks/zcash-test-vectors/commit/61894e7793d29fc236697ee187e486d84a31d158.